app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -139,6 +141,13 @@ private void setColumnNumber(int columns) {
     @Override
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
+
+        emptyView = new EmptyViewHandler(getContext());
+        emptyView.setIcon(R.attr.ic_folder);
+        emptyView.setTitle(R.string.no_subscriptions_head_label);
+        emptyView.setMessage(R.string.no_subscriptions_label);
+        emptyView.attachToListView(subscriptionGridLayout);
```

Comment:
I'd rather put this code into a new method `setupEmptyView`

Reply:
Done. Thanks for the review.